CMakeLists.txt

```diff
@@ -0,0 +1,134 @@
+# Copyright 2023 Viam Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: [required] libviam_rust_utils handling
+# TODO: [required] Dockerfile
```

Comment:
I wrote this TODO before I had taken a close look at the existing `Dockerfile`. I'm not actually sure there is actually anything actually TODO, because the current `Dockerfile` just builds third-party stuff.